formats/json-jackson/src/main/java/io/cloudevents/jackson/JsonFormat.java

```diff
@@ -220,4 +221,9 @@ static boolean dataIsJsonContentType(String contentType) {
         // If content type, spec states that we should assume is json
         return contentType == null || JSON_CONTENT_TYPE_PATTERN.matcher(contentType).matches();
     }
+
+    static boolean dataIsJsonContentType(ContentType contentType) {
+        // If content type, spec states that we should assume is json
+        return dataIsJsonContentType(contentType.value());
+    }
```

Comment:
where is this method used? It seems to be used only in tests?

Reply:
> …_where is this method used? It seems to be used only in tests?_…

A couple uses turn up in `/src/main/java`…

```bash
$ grep -R --include='*.java' 'dataIsJsonContentType' <path-to-sdk-java>
…
<path-to-sdk-java>/formats/json-jackson/src/main/java/io/cloudevents/jackson/CloudEventDeserializer.java:                                if (JsonFormat.dataIsJsonContentType(contentType)) {
<path-to-sdk-java>/formats/json-jackson/src/main/java/io/cloudevents/jackson/CloudEventDeserializer.java:                            if (JsonFormat.dataIsJsonContentType(contentType)) {
<path-to-sdk-java>/formats/json-jackson/src/main/java/io/cloudevents/jackson/CloudEventSerializer.java:                } else if (JsonFormat.dataIsJsonContentType(contentType)) {
<path-to-sdk-java>/formats/json-jackson/src/main/java/io/cloudevents/jackson/CloudEventSerializer.java:        if (JsonFormat.dataIsJsonContentType(contentType)) {
…
```

Reply:
The one with `ContentType contentType` as parameter? Aren't they using the one with `String contentType` as parameter?

Reply:
Ahh! OK. I'm with you now 👍

I should have paid closer attention to the method signature in your comment. It looks sorta familiar ;)